SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialLatencyTracer.h

```diff
@@ -35,6 +35,8 @@ class SPATIALGDK_API USpatialLatencyTracer : public UObject
 
 	//////////////////////////////////////////////////////////////////////////
 	//
+	// EXPERIMENTAL: This functionality is not currently supported for external users and shouldn't be used.
```

Comment:
```suggestion
	// EXPERIMENTAL: We do not support this functionality currently: Do not use it unless you are Improbable staff.
```

Something like that?

Suggest making it active voice so we are more direct. (It's the modern way in docs!)
Does anyone (apart from us) know what "external users"  are?

Reply:
I like your suggestion - just tagging @ernopp so we're all on the same page.

Reply:
LGTM thanks